.github/workflows/manual-deploy-dev-staging.yml

```diff
@@ -20,6 +20,29 @@ env:
   VETS_WEBSITE_CHANNEL_ID: C02V265VCGH # status-vets-website
 
 jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install dependencies
+        uses: ./.github/workflows/install
+        with:
+          key: ${{ hashFiles('yarn.lock') }}
+          yarn_cache_folder: .cache/yarn
+          path: |
+            .cache/yarn
+            node_modules
+
+      - name: Build dev
+        if: ${{ github.event.inputs.deploy_environment == 'dev' || github.event.inputs.deploy_environment == 'both' }}
+        run: yarn build --verbose —buildtype=dev
+        timeout-minutes: 30  
+        
+      - name: Build staging
+        if: ${{ github.event.inputs.deploy_environment == 'staging' || github.event.inputs.deploy_environment == 'both' }}
+        run: yarn build --verbose —buildtype=staging
+        timeout-minutes: 30 
```

Comment:
Same with this part, we need to update the `Deploy Dev` and `Deploy Staging` steps in the `Deploy` job for the behavior to change